my-app/src/frontend/components/navbar/Navbar.js

```diff
@@ -1,8 +1,38 @@
+import { FcSearch } from 'react-icons/fc';
+import { AiFillBook, AiFillHeart } from 'react-icons/ai';
+import { FaBookOpen, FaShoppingCart } from 'react-icons/fa';
+import "./navbar.css"
+
+
 
 
 function Navbar() {
   return (
-    <div>Navbar</div>
+    <nav class="navigation-container">
+    <div class="logo">
+    <FaBookOpen className='icon-size-small'/>
+    <span className="font-size-m">Kitab</span>
+    </div>
+
+    <div class="nav-search-bar">
+        <input type="search" placeholder="Search..." class="nav-search-input"></input>
+        <FcSearch/>
+    </div>
+    <div class="nav-options">
+        <div className='helloUser'>
+            <span>Hello User</span>
+            <button class="button-outlined font-size-s">Login</button>
```

Comment:
I think you are not maintaining consistency. use className everywhere to maintain consistency . it will help in code readability